python/lsst/alert/packet/updateSchema.py

```diff
@@ -0,0 +1,144 @@
+# This file is part of alert_packet.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import argparse
+import os
+import fastavro
+import yaml
+import json
+from lsst.alert.packet import get_schema_root, SchemaRegistry
+
+__all__ = ['update_schema']
+
+
+def update_docs(schema, apdb):
+    for index, schema_field in enumerate(schema['fields']):
```

Comment:
Rather than using an existing schema as a starting point, we want the apdb.yaml to be the source of truth.  That might require some mapping of data types between the yaml and Avro